keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -531,20 +532,30 @@ the kubelet, as described below. However, the source of this data may vary; for
 example, a cloud provider controller could populate this based upon information
 from the cloud provider API.
 
-In the kubelet case, each kubelet publishes kubelet publishes a set of
-`ResourceSlice` objects to the API server with content provided by the
-corresponding DRA drivers running on its node. Access control through the node
-authorizer ensures that the kubelet running on one node is not allowed to
-create or modify `ResourceSlices` belonging to another node. A `nodeName`
-field in each `ResourceSlice` object is used to determine which objects are
-managed by which kubelet.
-
-**NOTE:**  `ResourceSlices` are published separately for each driver, using
-whatever version of the `resource.k8s.io` API is supported by the kubelet. That
-same version is then also used in the gRPC interface between the kubelet and
-the DRA drivers providing content for those objects. It might be possible to
-support version skew (= keeping kubelet at an older version than the control
-plane and the DRA drivers) in the future, but currently this is out of scope.
+In the kubelet case, each driver running on a node publishes a set of
+`ResourceSlice` objects to the API server for its own resources, using its
+connection to the apiserver. Access control through a validating admission
```

Comment:
How does the VAP know which node's driver is acting, so that only the driver on node A can manipulate pools on node A?

Reply:
That is a weakness. We discussed this and the conclusion was that separation by driver would be nice, but not required because plugins are likely privileged on a node and thus a malicious or compromised driver can impersonate other drivers by stealing their credentials.Separation by driver would be a protection against bugs, but not a security boundary.

Reply:
From discussion, it seems like pod-bound SA tokens include node name, and VAP can see the token, so it SHOULD be able to verify that the driverName maps to the correct SA and/or namespace and that the nodeName maps to the calling pod's node.

I have to say that I dislike the "DIY" nature of this, but it seems like it will work.  Can we look for a more robust solution as we proceed?

Reply:
> That is a weakness. We discussed this and the conclusion was that separation by driver would be nice, but not required because plugins are likely privileged on a node and thus a malicious or compromised driver can impersonate other drivers by stealing their credentials.Separation by driver would be a protection against bugs, but not a security boundary.

I think Tim's original concern was not drivers overwriting other drivers' data, but a compromised node being able to overwrite other *nodes'* data. That is the more important item, though Tim's subsequent comment implies we may be able to do both.

I think we can resolve this comment.